source/fundamentals/builders/filters.txt

```diff
@@ -11,3 +11,542 @@ Filters Builders
+     - documents with the conditions of all the filters. This operator joins filters with a logical ``AND``.
+
+   * - :java-core-api:`or() <com/mongodb/client/model/Filters.html#or(org.bson.conversions.Bson...)>`
+     - documents with the conditions of either filter. This operator joins filters with a logical ``OR``.
+      
+   * - :java-core-api:`not() <com/mongodb/client/model/Filters.html#not(org.bson.conversions.Bson)>`
+     - documents that do not match the filter.
+     
+   * - :java-core-api:`nor() <com/mongodb/client/model/Filters.html#nor(org.bson.conversions.Bson...)>`
+     - documents that fail to match both filters. This operator joins filters with a logical ``NOR``.
+
+The following example creates a filter that matches documents where
+the value of the ``qty`` field is greater than "8" or the value
+of the ``color`` field equals "pink" in the ``paint_purchases`` collection: 
+
+.. literalinclude:: /includes/fundamentals/code-snippets/Filters.java
+   :language: java
+   :dedent:
+   :start-after: begin orComparison
+   :end-before: end orComparison
+   
+The JSON output renders as the following:
+
+.. code-block:: json
+   :copyable: false
+   
+   { "$or" : [
+         "qty" : { "$gt" : 8 }, 
+         "color" : "pink"
+         ] }
+
+
+The following shows the output of the query using the operator specified above:
+   
+.. code-block:: json
+   :copyable: false
+      
+   { "_id": 2, "color": "purple", "qty": 10, "vendor": ["C", "D"] }
+   { "_id": 5, "color": "yellow", "qty": 11, "vendor": ["A", "B"] }
+   { "_id": 6, "color": "pink", "qty": 5, "vendor": ["C"] }
+
+
+.. _arrays:
+
+Arrays
+------
+
+The array operators evaluate the array field in a document.
+
+The array operator methods include:
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - Array Method
+     - Matches
+      
+   * - :java-core-api:`all() <com/mongodb/client/model/Filters.html#all(java.lang.String,java.lang.Iterable)>`
+     - documents if the array field contains every element specified in the query.
+
+   * - :java-core-api:`elemMatch() <com/mongodb/client/model/Filters.html#elemMatch(java.lang.String,org.bson.conversions.Bson)>`
+     - documents if an element in the array field matches all the specified conditions.
+      
+   * - :java-core-api:`size() <com/mongodb/client/model/Filters.html#size(java.lang.String,int)>`
+     - documents if the array field is a specified number of elements.
+
+The following example matches documents with a ``vendors`` array
+containing both "A" and "D" in the ``paint_purchases`` collection: 
+
+.. literalinclude:: /includes/fundamentals/code-snippets/Filters.java
+   :language: java
+   :dedent:
+   :start-after: begin allComparison
+   :end-before: end allComparison
+   
+The JSON output renders as the following:
+
+.. code-block:: json
+   :copyable: false
+   
+   { "vendor" : { 
+      "$all" : { 
+         ["A", "D"] 
+         } } }
```

Comment:
Nitpick: this one is probably short enough to fit on one line. You can also close all the curly braces in a row without a space, e.g. "}}}"